src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.LoginExistException;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+        registrationService.register(new User("boblogin", "bobpassword", 18));
+        registrationService.register(new User("alicelogin", "alicepassword", 25));
+        registrationService.register(new User("denislogin", "denispassword",45));
+    }
+
+    @Test
+    void checkLoginEmpty_NotOk() {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/211330188-f97be301-d4a0-4467-a410-8c25f6db673a.png)

```suggestion
    void register_emptyLogin_NotOk() {
```